### ISSUE TITLE
RWD Bugs

### ISSUE THREAD

**Issue by ivanturlakov** [Joystream/pioneer#4609]
<img width="337" alt="Screenshot 2023-10-28 at 14 01 20" src="https://github.com/Joystream/pioneer/assets/46903215/84dbdda1-7248-4f71-8647-e8c5478ded40">
<img width="345" alt="Screenshot 2023-10-28 at 13 53 20" src="https://github.com/Joystream/pioneer/assets/46903215/40460500-fdb0-4a6c-9af6-f57ec1e32fcd">
<img width="345" alt="Screenshot 2023-10-28 at 13 50 21" src="https://github.com/Joystream/pioneer/assets/46903215/2af21544-a555-4e07-b9bb-308ca47cff5e">

<img width="797" alt="Screenshot 2023-10-27 at 21 42 12" src="https://github.com/Joystream/pioneer/assets/46903215/a400d950-e393-4bb3-bc9f-2c475eedf73d">
<img width="595" alt="Screenshot 2023-10-27 at 21 35 36" src="https://github.com/Joystream/pioneer/assets/46903215/3d3e9e86-2583-4c09-91b1-ec813b1f6bca">

Perhaps here we can hide the second column so that the third column is readable
<img width="608" alt="Screenshot 2023-10-27 at 22 21 28" src="https://github.com/Joystream/pioneer/assets/46903215/3fbfd452-e731-4d2d-a7f8-2cd9220a7959">

**Comment by ivanturlakov** [comment]
Also on mobile no chance to view tooltips(?)

**Comment by dmtrjsg** [comment]
@ivanturlakov great that you raised it as a separate issue. So the fixes also can come in separate PR instead of changing the existing HUGE pr 👍

**Comment by ivanturlakov** [comment]
@eshark9312 Please add a minor fix for Opening Form (stack it vertically - Question-100% wide, Answer-100% wide)
<img width="274" alt="Screenshot 2023-11-04 at 13 36 54" src="https://github.com/Joystream/pioneer/assets/46903215/6d1f9a62-89b5-433b-89b2-c8fc7d27257e">
<img width="333" alt="Screenshot 2023-11-04 at 13 36 24" src="https://github.com/Joystream/pioneer/assets/46903215/f17306e5-6ce8-4fbe-a4be-3bf4ccc6de75">

**Comment by chrlschwb** [comment]
@eshark9312 this issue was already addressed in https://github.com/Joystream/pioneer/pull/4607.
What is the status now? Will you create a separate PR?

**Comment by eshark9312** [comment]
@chrlschwb
Yes, addressed in #4607 and passed all QA tests.

**Comment by ivanturlakov** [comment]
@eshark9312 Looks like were missed this
<img width="843" alt="Screenshot 2023-11-06 at 13 34 18" src="https://github.com/Joystream/pioneer/assets/46903215/401cf9f0-171f-4365-9969-a6ec21cff223">

**Comment by eshark9312** [comment]
I've fixed it just before. Please check again.
https://dao-737dtg4ng-joystream.vercel.app/#/proposals/preview/626

**Comment by thesan** [comment]
> @chrlschwb Yes, addressed in #4607 and passed all QA tests.

As @dmtrjsg wrote in this https://github.com/Joystream/pioneer/issues/4609#issuecomment-1792264311 please address issues reported here in a separate PR, **not** in #4607 because this PR is starting to grow to big with changes unrelated to election, forum nor settings.

So until #4607 gets merged please address the QA feedback on a different branch based on `rwd/election,forum-and-settings` but that you can rebase after #4607 is merged.

**Comment by thesan** [comment]
The proposal empty state is no longer centered:
![image](https://github.com/Joystream/pioneer/assets/6571453/f91c1bf8-e2a3-4f48-965e-574c8c643323)

**Comment by thesan** [comment]
According to the new designs every pages main content should be in an at most `1150px` wide container:
![image](https://github.com/Joystream/pioneer/assets/6571453/01a327f1-4dcd-4c9a-82af-609a000de00e)

**Comment by thesan** [comment]
The members table headers are misaligned with the table content:
![image](https://github.com/Joystream/pioneer/assets/6571453/f4b582bb-7080-4465-9dd5-1c904af6d3f6)

**Comment by ivanturlakov** [comment]
Menu link active state
Cards layout
<img width="787" alt="Screenshot 2023-11-07 at 11 36 19" src="https://github.com/Joystream/pioneer/assets/46903215/c15f1b85-8275-49d5-a11d-3b02ba4bb57f">


Design
<img width="895" alt="Screenshot 2023-11-07 at 11 43 10" src="https://github.com/Joystream/pioneer/assets/46903215/0f36d64d-c684-4eda-86fb-1cae860ad55a">

**Comment by eshark9312** [comment]
@ivanturlakov Please check [here](https://dao-hrwt92kgz-joystream.vercel.app/).

**Comment by ivanturlakov** [comment]
@eshark9312 Looks good thank you!

One more thing
<img width="1438" alt="Screenshot 2023-11-09 at 14 11 57" src="https://github.com/Joystream/pioneer/assets/46903215/e3857e4f-5c20-4676-a461-0c74db2ec1dc">
<img width="1440" alt="Screenshot 2023-11-09 at 14 12 12" src="https://github.com/Joystream/pioneer/assets/46903215/230e5828-bd9a-469b-b1e4-2b6fd3f75a9d">

**Comment by eshark9312** [comment]
I fixed it.
https://dao-p59ur48rn-joystream.vercel.app/

**Comment by kdembler** [comment]
So so great to see this live and browse on mobile!

2 observations:
1. The content now doesn't take full page width anymore, I guess that's an expected change?
Before: https://share.cleanshot.com/W5427Ctb
After: https://share.cleanshot.com/J7fXnR4d
2. The selected tab underline now floats slightly above the bottom border
Before: https://share.cleanshot.com/r1WTJ4b9
After: https://share.cleanshot.com/WZdV6ytC

**Comment by dmtrjsg** [comment]
@kdembler looks like legit bugs to me, @eshark9312 would you like to address the comments from Klaudiusz as well?

**Comment by chrlschwb** [comment]
> So so great to see this live and browse on mobile!
> 
> 2 observations:
> 
> 1. The content now doesn't take full page width anymore, I guess that's an expected change?
>    Before: https://share.cleanshot.com/J7fXnR4d
>    After: https://share.cleanshot.com/W5427Ctb
> 2. The selected tab underline now floats slightly above the bottom border
>    Before: https://share.cleanshot.com/r1WTJ4b9
>    After: https://share.cleanshot.com/WZdV6ytC

1. That's an expected change.
2. I think that was missed during the QA. @eshark9312

**Comment by eshark9312** [comment]
@chrlschwb 
1. That's what is required from design.
2. I fixed this on https://github.com/Joystream/pioneer/pull/4636.
